tasks/build-monthly-updates-as-json.js

```diff
@@ -0,0 +1,121 @@
+    return gulp.src(filePattern)
+        .pipe(frontMatter({property: 'fm',remove: true}))
+        .pipe(tap(parseMarkdown))
+        .pipe(generateJson())
+        .pipe(jsonCombine(fileNameDestination, data => { return new Buffer.from(JSON.stringify(Object.values(data))); }))
+        .pipe(gulp.dest(fileDirectoryDestination));
+}
+
+function parseMarkdown(file) {
+    const env = {};
+    md.render(file.contents.toString(), env);
+    file.title = env.title;
+    file.anchorTitle = env.anchorTitle;
+    file.description = env.description;
+
+    return;
+}
+
+/**
+ * Return the first h1 title in the markdown file.
+ *
+ * Inspired by https://github.com/GerHobbelt/markdown-it-title/blob/master/index.js
+ */
+function parseTitleFromMarkdown(...args) {
+    const [ tokens, idx, , env, self ] = args;
+    let filteredTokens = tokens.filter(t => t.content.indexOf(':::') === -1);
+
+    if (!env.title && filteredTokens[idx].tag === 'h1') {
+        env.title = filteredTokens[idx + 1].content;
+        env.anchorTitle = filteredTokens[idx].attrs[0][1];
+    }
+
+    return self.renderToken(...args);
+};
+
+/**
+ * Return the description of the update feature.
+ *
+ * This description is the content of the paragraph just after the h1 title.
+ * If there is no paragraph after this title, the description is null.
+ */
+function parseDescriptionFromMarkdown(...args) {
+    const [ tokens, idx, , env, self ] = args;
+    let filteredTokens = tokens.filter(t => t.content.indexOf(':::') === -1);
+
+    const firsth1TitleIndex =  filteredTokens.findIndex((token) => token.tag === 'h1' && token.type === 'heading_close');
+    const tokensWithouth1 = filteredTokens.slice(firsth1TitleIndex);
+
+    const firsth2TitleIndex = tokensWithouth1.findIndex((token) => token.tag === 'h2');
+    const descriptionIndex = tokensWithouth1.findIndex((token) => token.type === 'inline');
+
+    if (descriptionIndex === -1) {
+        env.description = null;
+    } else if (firsth2TitleIndex === -1) {
+        env.description = tokensWithouth1[descriptionIndex].content;
+    } else if (descriptionIndex < firsth2TitleIndex) {
+        env.description = tokensWithouth1[descriptionIndex].content;
+    } else {
+        env.description = null;
+    }
+
+    return self.renderToken(...args);
+};
+
+
+function generateJson() {
+    return through((file, enc, cb) => {
+        // hardcoded to the 5th day of the month
+        let directoryName = path.basename(path.dirname(file.path));
+        let startDate = directoryName.replace('-', '/') + '/05';
+        let link = 'https://help.akeneo.com/pim/serenity/updates/' + directoryName + '.html#' + file.anchorTitle;
+
+        let defaultValues = {
+            'pim_announcement_img': null,
+            'pim_announcement_alt_img': null,
+            'pim_announcement_audience': [],
+        };
+
+        let data = {...defaultValues, ...file.fm };
+
+        let content = JSON.stringify({
```

Comment:
It will needs to have endDate also in the json exported as we will need it to know if it will have a "New" style  : https://github.com/akeneo/actionable-product-data/issues/132

Reply:
It's the duration, we can calculate it in the proxy. For Tata, it's easier to work with duration (X days of notification).